tests/Admin/Extension/LockExtensionTest.php

```diff
@@ -235,7 +235,7 @@ private function configureFormMapper(): FormMapper
     private function configureAdmin(
         string $uniqid = '',
         ?Request $request = null,
-        ?ModelManagerInterface $modelManager = null
+        ModelManagerInterface $modelManager
```

Comment:
Can you change the order of the param to 
`$modelManager, $uniqId, $request`.

This allow `$this->configureAdmin($this->modelManager)` instead of `$this->configureAdmin('', null, $this->modelManager)`